monitoring/prometheus/alerts/ceph_default_alerts.yml

```
@@ -214,6 +214,17 @@ groups:
             will be full in less than 5 days assuming the average fill-up
             rate of the past 48 hours.
 
+      - alert: MTU Mismatch
+        expr: node_network_mtu_bytes{device!="lo"} != on() group_left() quantile(0.5, node_network_mtu_bytes{device!="lo"})
+        labels:
+          severity: warning
+          type: ceph_default
+          oid: 1.3.6.1.4.1.50495.15.1.2.8.5
+        annotations:
+          description: >
+            Node {{ $labels.instance }} has a different MTU size than majority of nodes
+            on device {{ $labels.device }}.
```

What about printing the value and mentioned, as @p-se suggested to call it 'median' rather than majority...
```suggestion
            Node {{ $labels.instance }} has a different MTU size ({{ $value }})
            than the median value on device {{ $labels.device }}.
```